store/views.py

```diff
@@ -77,6 +105,52 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    if request.method == 'POST':
+        book_id = request.POST['id']
+        try:
+            book = BookCopy.objects.get(pk=book_id)
+        except:
+            return JsonResponse({'message':'Book not found'})  
+
+        else:
+            book.status = True
+            book.borrower = None
+            book.borrow_date = None
+            book.save()
+            return JsonResponse({'message':'success'})
+    else:
+        return JsonResponse({'message':'Invalid request method'})
+@csrf_exempt
+@login_required
+def rateBookView(request):
+    if request.method == "POST":
+        bid = request.POST['bid']
+        new_rating = Decimal(request.POST['rating'])
+        if new_rating >= 0 and new_rating <= 10:
+            try:
+                book = Book.objects.get(pk=bid)
+                user = User.objects.get(username = request.user.username)
+                previous_user_rating = BookRating.objects.filter(book = book, user = user)
+                previous_user_rating.delete()
+                obj = BookRating()
+                obj.user = user
+                obj.book = book
+                obj.rate = new_rating
+                obj.save()
+                books = BookRating.objects.filter(book = book)
+                total = 0
+                for i in books:
+                    total+=i.rate
+                book.rating = total/books.count()
+                book.save()
+            except:
```

Comment:
As a good coding practice, whenever you use try-except block, capture only the exceptions which you want to catch (IndexError, IntegrityError, etc.)